> > macOS is failing with:
> > ```
> > FAILED: gazebo/gui/CMakeFiles/gazebo_gui.dir/qgv/private/QGVCore.cpp.o 
> > /Users/runner/miniconda3/envs/test/bin/x86_64-apple-darwin13.4.0-clang++ -DBOOST_ALL_NO_LIB -DBOOST_DATE_TIME_DYN_LINK -DBOOST_FILESYSTEM_DYN_LINK -DBOOST_IOSTREAMS_DYN_LINK -DBOOST_PROGRAM_OPTIONS_DYN_LINK -DBOOST_REGEX_DYN_LINK -DBOOST_SYSTEM_DYN_LINK -DBOOST_THREAD_DYN_LINK -DBUILDING_DLL -DBUILDING_DLL_GZ_GUI -DHAVE_OPENGL -DHAVE_STRING_H -DIGN_PROFILER_ENABLE=0 -DLIBBULLET_VERSION=3.20 -DLIBBULLET_VERSION_GT_282 -DMAC_OS_X_VERSION=1080 -DQGVCORE_LIB -DQT_CORE_LIB -DQT_GUI_LIB -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQWT_DLL -DWITH_CGRAPH -Dgazebo_gui_EXPORTS -I/Users/runner/work/gazebo-classic/gazebo-classic/build/gazebo/gui/gazebo_gui_autogen/include -I/Users/runner/work/gazebo-classic/gazebo-classic/test/gtest/include -I/Users/runner/work/gazebo-classic/gazebo-classic -I/Users/runner/work/gazebo-classic/gazebo-classic/build -I/Users/runner/work/gazebo-classic/gazebo-classic/deps -I/Users/runner/work/gazebo-classic/gazebo-classic/gazebo/gui/qgv -I/Users/runner/work/gazebo-classic/gazebo-classic/gazebo/gui/qgv/private -I/Users/runner/miniconda3/envs/test/include/libusb-1.0 -I/Users/runner/miniconda3/envs/test/include/graphviz -isystem /Users/runner/miniconda3/envs/test/include/ignition/msgs5 -isystem /Users/runner/miniconda3/envs/test/include/ignition/math6 -isystem /Users/runner/miniconda3/envs/test/include/ignition/transport8 -isystem /Users/runner/miniconda3/envs/test/include/ignition/common3 -isystem /Users/runner/miniconda3/envs/test/include/ignition/fuel_tools4 -iframework /Applications/Xcode_14.0.1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks -isystem /Users/runner/miniconda3/envs/test/include/OGRE/RTShaderSystem -isystem /Users/runner/miniconda3/envs/test/include/OGRE -isystem /Users/runner/miniconda3/envs/test/include/OGRE/OSX -isystem /Users/runner/miniconda3/envs/test/include/OGRE/Terrain -isystem /Users/runner/miniconda3/envs/test/include/OGRE/Paging -isystem /Users/runner/miniconda3/envs/test/include/OGRE/Overlay -isystem /Users/runner/miniconda3/envs/test/include/sdformat-9.8 -isystem /Users/runner/miniconda3/envs/test/include/qt -isystem /Users/runner/miniconda3/envs/test/include/qt/QtCore -isystem /Users/runner/miniconda3/envs/test/./mkspecs/macx-clang -isystem /Users/runner/miniconda3/envs/test/include/sdformat-9.8/sdf/.. -isystem /Users/runner/miniconda3/envs/test/include/ignition/cmake2 -isystem /Users/runner/miniconda3/envs/test/include/qt/QtWidgets -isystem /Users/runner/miniconda3/envs/test/include/qt/QtGui -isystem /Applications/Xcode_14.0.1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/OpenGL.framework/Headers -march=core2 -mtune=haswell -mssse3 -ftree-vectorize -fPIC -fPIE -fstack-protector-strong -O2 -pipe -stdlib=libc++ -fvisibility-inlines-hidden -fmessage-length=0 -isystem /Users/runner/miniconda3/envs/test/include      -stdlib=libc++  -Wall -Wextra -Wno-long-long -Wno-unused-value -Wfloat-equal -Wshadow -Wswitch-default -Wmissing-include-dirs -pedantic    -O3 -DNDEBUG -mssse3 -msse3 -msse -msse2  -isysroot /Applications/Xcode_14.0.1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk -fPIC   -fPIC -fPIC -std=c++17 -MD -MT gazebo/gui/CMakeFiles/gazebo_gui.dir/qgv/private/QGVCore.cpp.o -MF gazebo/gui/CMakeFiles/gazebo_gui.dir/qgv/private/QGVCore.cpp.o.d -o gazebo/gui/CMakeFiles/gazebo_gui.dir/qgv/private/QGVCore.cpp.o -c /Users/runner/work/gazebo-classic/gazebo-classic/gazebo/gui/qgv/private/QGVCore.cpp
> > In file included from /Users/runner/work/gazebo-classic/gazebo-classic/gazebo/gui/qgv/private/QGVCore.cpp:19:
> > /Users/runner/work/gazebo-classic/gazebo-classic/gazebo/gui/qgv/private/QGVCore.h:99:17: error: no member named 'putstr' in 'Agiodisc_s'
> >       memIoDisc.putstr = AgIoDisc.putstr;
> >       ~~~~~~~~~ ^
> > /Users/runner/work/gazebo-classic/gazebo-classic/gazebo/gui/qgv/private/QGVCore.h:99:35: error: no member named 'putstr' in 'Agiodisc_s'
> >       memIoDisc.putstr = AgIoDisc.putstr;
> >                          ~~~~~~~~ ^
> > 2 errors generated.
> > [855/1158] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/plot/VariablePill.cc.o
> > [856/1158] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/plot/PlotWindow.cc.o
> > [857/1158] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/plot/VariablePillContainer.cc.o
> > [858/1158] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/plot/TopicCurveHandler.cc.o
> > ninja: build stopped: subcommand failed.
> > ```
> 
> This is because I never merged #3270 . -_-

Actually, the problem is that for some reason on macos graphviz 4 is installed, while graphviz 7 is available.

--- a/.github/workflows/conda-forge.yml
+++ b/.github/workflows/conda-forge.yml
@@ -25,9 +25,8 @@ jobs:
 
     - uses: conda-incubator/setup-miniconda@v2
       with:
-        mamba-version: "*"
-        channels: conda-forge
-        channel-priority: true
+        miniforge-variant: Mambaforge
+        miniforge-version: latest
 
     - name: Dependencies
       shell: bash -l {0}

--- a/gazebo/gui/CMakeLists.txt
+++ b/gazebo/gui/CMakeLists.txt
@@ -160,6 +160,17 @@ if (HAVE_GRAPHVIZ)
   # Activate Export macro for building the library itself
   # if QGVCORE_LIB is not set or equal 0 (null), import macro is used
   add_definitions(-DQGVCORE_LIB)
+  # If Graphviz >= 3.0.0 is used as shared library on Windows, the GVDLL 
+  # preprocess definition should be defined. As at this point we can't 
+  # know if graphviz is shared or static, we always defined GVDLL, 
+  # leaving an option that users can use to explicitly specify that 
+  # the linked graphviz is static
+  option(GAZEBO_FINDGRAPHVIZ_USE_STATIC_GRAPHVIZ 
+         "Enable when linking a static graphviz" OFF)
+  mark_as_advanced(GAZEBO_FINDGRAPHVIZ_USE_STATIC_GRAPHVIZ)
+  if(WIN32 AND NOT GAZEBO_FINDGRAPHVIZ_USE_STATIC_GRAPHVIZ)
+    add_definitions(-DGVDLL)
+  endif()
   include_directories(${GRAPHVIZ_INCLUDE_DIR})
   include_directories(${CMAKE_CURRENT_SOURCE_DIR}/qgv)
   include_directories(${CMAKE_CURRENT_SOURCE_DIR}/qgv/private)

